routes/web.php

```diff
@@ -94,15 +95,34 @@
         Route::post('/genres/store', [GenreController::class, 'store'])->name('genres.store');
         Route::patch('/genres/{genre}/update', [GenreController::class, 'update'])->name('genres.update');
         Route::delete('/genres/{genre}/destroy', [GenreController::class, 'destroy'])->name('genres.destroy');
-        
+        // ADMIN-RECOMMENDED PLAN
+        Route::get('/recommended/plans',[RecommendedPlanController::class, 'index'])->name('recommended_plans');
+        Route::post('/recommended/plans/store', [RecommendedPlanController::class, 'store'])->name('recommended_plans.store');
+        Route::patch('/recommended/plans/{recommended_plans}/update', [RecommendedPlanController::class, 'update'])->name('recommended_plans.update');
```

Comment:
I suggest to use `{plan}` here instead of `{recommended_plans}` here so that in your `RecommendedPlanController` we can easily inject `Plan` model into your update function. 
your `update($id, Request $request)` function in the controller will change to `update(Plan $recommended_plan, Request $request)` and there's no need for `Plan::findOrFail($id);` in your update function